### PR TITLE
Web paths should always replace \ with /

### PR DESCRIPTION
**Pull request by TheBosZ**
Related to #36

### CODE REVIEW

File: lib/src/device.dart [hunk 1]
```diff
@@ -775,7 +775,7 @@ class AndroidDevice extends Device {
       runCheckedSync([adbPath, 'reverse', serverPortString, serverPortString]);
     }
 
-    String relativeDartMain = path.relative(mainDart, from: serverRoot);
+    String relativeDartMain = path.relative(mainDart, from: serverRoot).replaceAll('\\', '/');
```
Comment (abarth) [inline lib/src/device.dart]: I would probably move this into a private top-level function called something like `_convertToURl` to explain why we're making this replacement.

**Comment by abarth** [comment]
LGTM

File: lib/src/device.dart (hunk 1 above)
Comment (TheBosZ) [inline lib/src/device.dart]: I thought I had put in a comment explaining it, but I can make a function, no problem.

**Comment by TheBosZ** [comment]
Changed to use a _convertToURL function to be more explicit

**Comment by abarth** [comment]
LGTM

**Pull request merged**